tox.ini

```diff
@@ -14,7 +14,6 @@ tst_path = {toxinidir}/tests/
 all_path = {[vars]src_path} {[vars]tst_path}
 
 [testenv]
-basepython = python3.10
```

Comment:
Is this needed?

Reply:
For local development (as I have been on noble, which ships python 3.12 per default). The question would be if there is a benefit in keeping it.